src/Ng/Jobs/Monitoring2MonitoringJob.cs

```diff
@@ -42,6 +46,13 @@ protected override void Init(IDictionary<string, string> arguments, Cancellation
 
         protected override async Task RunInternal(CancellationToken cancellationToken)
         {
+            var currentMessageCount = await _queue.GetMessageCount(cancellationToken);
+            if (currentMessageCount > _maxRequeueQueueSize)
+            {
+                Logger.LogInformation("Can't requeue any invalid packages because the queue has too many messages!");
```

Comment:
>LogInformation [](start = 23, length = 14)

Also, it could be useful to log the value of `currentMessageCount`

Reply:
I'll log both the current message count and the max size, good idea!

Reply:
also, would it be easier to log the count of `invalid` blobs somewhere in AI when not requeueing? don't need to go to the blob storage to look it up?